app/models/track.py

```diff
@@ -9,19 +9,21 @@ class Track(db.Model):
     description = db.Column(db.Text)
     track_image_url = db.Column(db.Text)
     color = db.Column(db.String, nullable=False)
+    font_color = db.Column(db.String)
```

Comment:
Don't store fontcolor to data base. It's a generated property. Generate it based on track color and provide it in the API